### ISSUE TITLE
exc_info is false

### ISSUE THREAD

**Issue by shizacat** [keeprocking/pygelf#29]
Hello.
The error appear in this place: https://github.com/keeprocking/pygelf/blob/master/pygelf/gelf.py#L40
```
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/handlers.py", line 621, in emit
    s = self.makePickle(record)
  File "/usr/local/lib/python3.5/dist-packages/pygelf/handlers.py", line 86, in makePickle
    return self.convert_record_to_gelf(record)
  File "/usr/local/lib/python3.5/dist-packages/pygelf/handlers.py", line 34, in convert_record_to_gelf
    gelf.make(record, self.domain, self.debug, self.version, self.additional_fields, self.include_extra_fields),
  File "/usr/local/lib/python3.5/dist-packages/pygelf/gelf.py", line 41, in make
    gelf['full_message'] = '\n'.join(traceback.format_exception(*record.exc_info))
TypeError: format_exception() argument after * must be an iterable, not bool
```
In documentation:

> If exc_info does not evaluate as false, it causes exception information to be added to the logging message. If an exception tuple (in the format returned by sys.exc_info()) or an exception instance is provided, it is used; otherwise, sys.exc_info() is called to get the exception information.

May be add check?

**Comment by keeprocking** [comment]
Yep, will do.

**Comment by bmerry** [comment]
I've submitted a PR for this. In the meantime I'm using the following workaround:

```python
class FixExcInfo(object):
    def filter(self, record):
        if not record.exc_info:
            record.exc_info = None
        return record

...
handler.addFilter(FixExcInfo())
```

where `handler` is one of the Gelf handlers.

**Comment by keeprocking** [comment]
Fixed in `0.3.6`, many thanks to @bmerry
